src/hybrid_a_star_planner/path_planner/collision_check.py

```
@@ -60,15 +60,43 @@ def check_collision(traj: List[List[float]], map_params: MapParameters) -> bool:
 
     return False
 
+def is_collision_with_dynamic_obstacles(
+    traj: List[tuple],
+    map_params: MapParameters,
+    start_time: float = 0.0,
+    dt: float = 0.2,
+    threshold: float = 2.0  #阈值越大，越容易判断为碰撞，离障碍物远远的，这里根据栅格大小以及车辆的大小设为栅格的1/2
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)